test/elm.js

```diff
@@ -0,0 +1,14 @@
+const assert = require('assert');
+const {bundle, run} = require('./utils');
+
+describe('elm', function() {
+  it('should produce a bundle', async function() {
+    let b = await bundle(__dirname + '/integration/elm/index.js');
```

Comment:
Oh. This won't work. Create an index.html file that looks like this:

```
<html>
<body>
<script src="./index.js"></script>
</body>
</html>
```

Then change that line to

```
 +    let b = await bundle(__dirname + '/integration/elm/index.html');
```

Reply:
Ignore that, sorry.